src/components/Editor/Tab.js

```diff
@@ -14,6 +14,7 @@ import CloseButton from "../shared/Button/Close";
 
 import type { List } from "immutable";
 import type { SourceRecord } from "../../reducers/sources";
+import type { SourceMetaDataType } from "../../reducers/ast";
```

Comment:
Maybe types should have their own folder instead of being together with reducers  ... but that's for another issue I guess.